src/crypto/mod.rs

```diff
@@ -1056,6 +1073,7 @@ impl Serialize for COSEKey {
 
 /// Errors that can be returned from COSE functions.
 #[derive(Debug, Clone, Serialize)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
I think we can make this struct just PartialEq all the time

Reply:
Done! https://github.com/mozilla/authenticator-rs/pull/337/commits/093957b831de339a0b971f91f6aa560353c0ab62